actions.yaml

```diff
@@ -4,10 +4,10 @@
 get-primary:
   description: Report primary replica
 
-get-admin-password:
+get-password:
   description: Get the admin user password used by charm.
 
-set-admin-password:
+set-password:
   description: Change the admin user password used by charm.
   params:
     password:
```

Comment:
In Kafka , the signature for set-password also has a `username` parameter

https://github.com/canonical/kafka-operator/blob/main/actions.yaml#L4

Should we try to get a unified UX experience for parameters too?

Probably it would be best to get the same description across

Reply:
Agreed about the description - but I don't think it makes sense if we only have the admin user in MongoDB to have a user parameter. What do you think @deusebio

Reply:
Oh, I see. Probably it is really not possible to get a fully unified UX. Let's see whether this is really something we want to get at a later stage.